pages/workspace/inspector.mdx

```diff
@@ -0,0 +1,232 @@
+---
+layout: 'guide'
+title: 'Inspector'
+# abstract: 'Learn about how to use the inspector dashboard'
+---
+
+import Link from '../../components/Link';
+
+The inspector tab is where you configure your inspector sources, get API keys to send data form the clients, set up slack alerts and examine the issues that Inspector found.
+
+### Default view and switching between views
+
+If you have no sources set up you will thee the <Link passHref href="./inspector#manage-sources"><a>Sources screen</a></Link> as the default view of the Inspector Tab.
+If you already have set up a source you would see the <Link passHref href="./inspector#view-events"><a>Events screen</a></Link>.
+Use the button in the top left corner to switch between the Sources and the Events screens.
+
+## Manage Sources
+
+### Sources
+
+Inspector shares sources with the <Link passHref href="/workspace/connections"><a>Connections</a></Link> tab.
+You'll see a grid of all sources set up in Avo in the connections tab.
+You can create new Inspector sources here, it will also appear in the Connections Tab.
+
+Source card have 3 states:
+
+- Receiving events - this source already received at least one event from the client SDK.
+
+<center>
+  <img
+    src={require('../../images/workspace/inspector/inspector-source-receiving.png')}
+    alt="Opening a branch"
+  />
+</center>
```

Comment:
Could we reduce the size of these screenshots? Would be nice if we could keep them inline aligned to the right 👀

Reply:
How would I do that?

Reply:
I think something like `width="250"` on the img tag would do